### ISSUE TITLE
Config errors and no requests / traffic reporting after changing the logger

### ISSUE THREAD

**Issue by roman-kruglov** [Arriven/db1000n#513]
tons of errors like:
```
{"level":"error","ts":1650644178.083019,"caller":"job/runner.go:193","msg":"error running job","name":"","type":"encrypted","error":"no identity matched any of the recipients","stacktrace":"github.com/Arriven/db1000n/src/job.(*Runner).runJobs.func1\n\t/home/runner/work/db1000n/db1000n/src/job/runner.go:193"}
{"level":"error","ts":1650644178.515815,"caller":"job/runner.go:193","msg":"error running job","name":"","type":"encrypted","error":"no identity matched any of the recipients","stacktrace":"github.com/Arriven/db1000n/src/job.(*Runner).runJobs.func1\n\t/home/runner/work/db1000n/db1000n/src/job/runner.go:193"}
{"level":"error","ts":1650644178.9551759,"caller":"job/runner.go:193","msg":"error running job","name":"","type":"encrypted","error":"no identity matched any of the recipients","stacktrace":"github.com/Arriven/db1000n/src/job.(*Runner).runJobs.func1\n\t/home/runner/work/db1000n/db1000n/src/job/runner.go:193"}
{"level":"error","ts":1650644179.3871958,"caller":"job/runner.go:193","msg":"error running job","name":"","type":"encrypted","error":"no identity matched any of the recipients","stacktrace":"github.com/Arriven/db1000n/src/job.(*Runner).runJobs.func1\n\t/home/runner/work/db1000n/db1000n/src/job/runner.go:193"}
{"level":"error","ts":1650644179.82688,"caller":"job/runner.go:193","msg":"error running job","name":"","type":"encrypted","error":"no identity matched any of the recipients","stacktrace":"github.com/Arriven/db1000n/src/job.(*Runner).runJobs.func1\n\t/home/runner/work/db1000n/db1000n/src/job/runner.go:193"}
{"level":"error","ts":1650644180.274406,"caller":"job/runner.go:193","msg":"error running job","name":"","type":"encrypted","error":"no identity matched any of the recipients","stacktrace":"github.com/Arriven/db1000n/src/job.(*Runner).runJobs.func1\n\t/home/runner/work/db1000n/db1000n/src/job/runner.go:193"}
```
and no requests / traffic reporting showing up anymore. Version - latest, 28.

P.S. I got the same config errors on .25 version

**Comment by maksim-sidorenko** [comment]
Kind of the same for me as well
```txt
{"level":"error","ts":1650643872.5228965,"caller":"utils/utils.go:20","msg":"caught panic, recovering","err":"runtime error: invalid memory address or nil pointer dereference","stacktrace":"github.com/Arriven/db1000n/src/utils.PanicHandler\n\tgithub.com/Arriven/db1000n/src/utils/utils.go:20\nruntime.gopanic\n\truntime/panic.go:838\nruntime.panicmem\n\truntime/panic.go:220\nruntime.sigpanic\n\truntime/signal_unix.go:818\ngithub.com/Arriven/db1000n/src/utils/metrics.(*Accumulator).Clone\n\tgithub.com/Arriven/db1000n/src/utils/metrics/metrics.go:231\ngithub.com/Arriven/db1000n/src/job.parallelJob\n\tgithub.com/Arriven/db1000n/src/job/complex.go:103\ngithub.com/Arriven/db1000n/src/job.encryptedJob\n\tgithub.com/Arriven/db1000n/src/job/utils.go:249\ngithub.com/Arriven/db1000n/src/job.(*Runner).runJobs.func1\n\tgithub.com/Arriven/db1000n/src/job/runner.go:192"}
```

**Comment by matherfather** [comment]
Have the same cloud provider + protonvpn , manual

**Comment by Arriven** [comment]
All of those should be fixed in 0.8.29

**Comment by roman-kruglov** [comment]
I think these particular defects are addressed and fixed, closing the issue
